src/datamodel_code_generator/util.py

```
@@ -66,24 +82,43 @@ def _construct_yaml_bool_with_warning(loader: Any, node: Any) -> bool:
-        "tag:yaml.org,2002:bool",
-        _YAML_1_2_BOOL_PATTERN,
-    ))
-SafeLoaderTemp.add_constructor("tag:yaml.org,2002:bool", _construct_yaml_bool_with_warning)
-
-SafeLoader = SafeLoaderTemp
-
-Model = TypeVar("Model", bound=_BaseModel)
+@lru_cache(maxsize=1)
+def get_safe_loader() -> type:
+    """Get customized SafeLoader lazily."""
+    import copy  # noqa: PLC0415
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)